frontend/packages/ceph-storage-plugin/integration-tests/tests/pvc.scenario.ts

```diff
@@ -0,0 +1,56 @@
+import { browser } from 'protractor';
+import { appHost } from '@console/internal-integration-tests/protractor.conf';
+import * as crudView from '@console/internal-integration-tests/views/crud.view';
+import {
+  createNewPersistentVolumeClaim,
+  deletePersistentVolumeClaim,
+  goToPersistentVolumeClaims,
+} from '../views/pvc.view';
+
+describe('Test PVC creation with options.', () => {
+  beforeAll(async () => {
+    await browser.get(`${appHost}/`);
+  });
+
+  it('Test RBD PVC is created and gets bound', async () => {
+    await createNewPersistentVolumeClaim(
+      'rbdpvc',
+      'openshift-storage',
+      '3',
+      'ocs-storagecluster-ceph-rbd',
+      'ReadWriteOnce',
+      true,
+    );
+    await goToPersistentVolumeClaims();
+    await crudView.resourceRowsPresent();
+    await deletePersistentVolumeClaim('rbdpvc', 'openshift-storage');
```

Comment:
What are we verifying here? I don't see any `expect` statement.

Reply:
Actually, I do verify the status inside the function for PVC creation, the last parameter is whether to wait for Bound status or not. I wonder how to make it more obvious, though.

Reply:
I can see now.  I feel these checks should be done via `expect` so that if it fails, we should know the reason. As per this implementation, it will time out and will not communicate the correct reason. WDYT?

Reply:
Added explicit status checks to the test functions.